interface/forms/questionnaire_assessments/questionnaire_assessments.php

```diff
@@ -112,11 +112,16 @@
 /* where to put the LOINC statement */
 $top_note = $bottom_note = false; // value of '3' says don't display
 
-    switch ($GLOBALS['questionnaire_display_LOINCnote']){
-       case '0': $top_note = true; break;
-       case '1': $bottom_note = true; break;
-       case '2': $top_note = $bottom_note = true;
-    }
+switch ($GLOBALS['questionnaire_display_LOINCnote']) {
+    case '0':
+            $top_note = true;
+             break;
+    case '1':
+            $bottom_note = true;
+             break;
+    case '2':
+            $top_note = $bottom_note = true;
+}
```

Comment:
indent 4 spaces for each level so case '?': level is correct however last level(setting var) is 8 spaces instead of 4 i.e. $top_notes '$' should be under the space after case.
```
case '0':
    $top_note` = true;
    break;
```

Reply:
ah thanks jerry, i'll amend and resubmit - it's not urgent btw

Reply:
i'm just improving the code - will do another PR shortly...